pubspec.yaml

```diff
@@ -1,8 +1,8 @@
-name: test_coverage
+name: test_coverage_advanced
```

Comment:
I must have missed it somewhere... why are we changing the package name here?

Reply:
i had problems with cache - fixed for now ;) sorry